tests/travis/install-db2.sh

```diff
@@ -4,19 +4,20 @@ set -ex
 
 echo Setting up IBM DB2
 
-sudo docker pull ibmcom/db2express-c:10.5.0.5-3.10.0
+echo "su - db2inst1 -c 'db2 CONNECT TO doctrine && db2 CREATE USER TEMPORARY TABLESPACE doctrine_tbsp PAGESIZE 4 K'" > /tmp/doctrine-init.sh
+chmod +x /tmp/doctrine-init.sh
+
 sudo docker run \
     -d \
     -p 50000:50000 \
     -e DB2INST1_PASSWORD=Doctrine2018 \
     -e LICENSE=accept \
+    -e DBNAME=doctrine \
+    -v /tmp/doctrine-init.sh:/var/custom/doctrine-init.sh:ro \
     --name db2 \
-    ibmcom/db2express-c:10.5.0.5-3.10.0 \
-    db2start
-
-sleep 15
+    --privileged=true \
+    ibmcom/db2
 
-sudo docker exec db2 su - db2inst1 -c \
-    'db2 CREATE DB doctrine && db2 CONNECT TO doctrine && db2 CREATE USER TEMPORARY TABLESPACE doctrine_tbsp PAGESIZE 4 K'
+sudo docker logs -f db2 | sed '/(*) Setup has completed./ q'
```

Comment:
Is it possible to use `dockerize` instead?

Reply:
I haven't used it before. What would be the benefit and what would the command look like?

Reply:
The [documentation](https://hub.docker.com/r/ibmcom/db2#quick-start) says:

> To confirm Db2 container is ready, in the logs we will see the message **Setup has completed**.

While we can ping the container until it's ready, but our experience with MySQL shows that often it's not reliable.